support-workers/src/test/scala/com/gu/support/workers/lambdas/FailureHandlerSpec.scala

```diff
@@ -117,8 +117,6 @@ class FailureHandlerITSpec extends AsyncLambdaSpec with MockContext {
 
     val testFields = FailedEmailFields.digitalPack("test@thegulocal.com", IdentityUserId("30001643"))
 
-    when(emailService.send(any[EmailFields])).thenReturn(Future.successful(()))
```

Comment:
this returns Unit now [1], so there's no need to tell it to return anything specific

[1] https://github.com/guardian/support-frontend/pull/5850/files#diff-54f1a2c774e69ee7474fea35d7431f5dbc1d707ced5a0b32006ebddc7612915fR26